index.html

```diff
@@ -16,16 +15,17 @@
         }
     </style>
 
-    <script defer>
-      window.snake.start();
-    </script>
-
 </head>
 
 
 <body>
 
 
 </body>
+<script defer src="./src/snake.min.js">
+
+ window.snake.start();
+
+</script>
```

Comment:
There's no point to putting the script block at the end.  This is a dead 1990s strategy.

The original idea was that it would get your script to run after the HTML, but `defer`, which you're already doing, is the vastly better way to accomplish that

Some people would argue for the lower load impact, except it's coming from a second HTTP stream (it's not inline,) so if anything this causes a trivial slowdown

Most importantly, the HTML spec expects `<script>` to be in `<head>`